.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ jobs:
       ignore:
         - gh-pages
     docker:
-      - image: python:3.7
+      - image: python:3.8
```

Comment:
Are the changes in the circleci config needed to get rid of the warnings?

Reply:
The markdown warning only happens when using python 3.7 or lower, as far as I can tell